ballerina/modules/apex/Module.md

```diff
@@ -0,0 +1,98 @@
+## Overview
+
+Salesforce Apex REST API enables you to expose your Apex classes and methods as RESTful web services. This module provides operations for executing custom Apex REST endpoints, allowing you to perform various HTTP operations on these endpoints and handle responses accordingly.
```

Comment:
Wouldn't it be better to mention the Apex API version we currently support?

Reply:
it is here: the same functionality., Added in here also.